semcore/popper/src/Popper.jsx

```diff
@@ -44,6 +44,13 @@ const useUpdatePopperEveryFrame = (popperRef) => {
   return handleAnimationFrame;
 };
 
+const hasParent = (element, parent) => {
```

Comment:
you has `hasParent` method here - https://github.com/semrush/intergalactic/pull/1075/files#diff-bc07519d4d3dfe3c56afead9beb044a989b3b60089dca88810bf7d6162f3d545

Reply:
Imported it from utils